### ISSUE TITLE
Change sampling mode in runtime

### ISSUE THREAD

**Issue by AndreYin** [Scirra/Construct-feature-requests#347]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

It would be great to have an option to change sampling mode in runtime. I'm making a low-res pixel art game and so I've been using the nearest sampling option, which looks great, but in some areas I would need the sampling rate to be linear - for instance when showing a fullscreen video.

### Possible workarounds or alternatives

There's no workaround currently.

### Proposed solution

Ideally we should have an action to change sampling mode during runtime.

### Why is this idea important?

This has been asked for a long time, and it was supposed to be extremely hard with webgl1 as seen here: https://www.construct.net/en/forum/construct-3/general-discussion-7/changing-project-sampling-175787

**Comment by F3der1co** [comment]
Maybe the time has come to ditch webgl1, weGPU so far has been an absolute let down. So upgrading the webgl renderer to make full use of ES3 features sounds like a good idea.

**Comment by MikalDev** [comment]
Agree, good point. Or at least enable some features for webgl2 that may not be enabled for webgl2 (e.g. have no effect.)

**Comment by alastajj** [comment]
Yeah holding back an important feature because it doesn't work in WebGL 1, sounds like a bad idea. 

Surely all relevant Steam customers would be able to run a WebGL2 game.
